src/classes/STG_PanelDataImportAdvancedMapping_CTRL.cls

```diff
@@ -128,6 +128,18 @@ public with sharing class STG_PanelDataImportAdvancedMapping_CTRL extends STG_Pa
         setDeploymentStatus();
     }
 
+    /*******************************************************************************************************
+    * @description Action Method to navigate to separate VF page that contains custom Advanced Mapping UI in
+    * Aura and Lightning
+    *
+    * @return PageReference
+    */
+    public PageReference navigate(){
+        PageReference advancedMapping = new PageReference('/lightning/cmp/c__BDI_ManageAdvancedMapping');
```

Comment:
Is there a better way to get this string? Or at least define as constant someplace more visible?